src/content/en/fundamentals/payments/payment-apps-developer-guide/web-payment-apps.md

```diff
@@ -421,9 +421,12 @@ Following are the conditions to enable skip-the-sheet:
 
 1. Only one payment method identifier is specified in the `PaymentRequest`
```

Comment:
This condition is not needed anymore.

Regardless of how many payment method identifiers is specified as long as there is only a single payment handler available with full delegation Chrome would skip to the payment handler if show() is triggered with a user gesture.

Reply:
Fixed.